src/main/java/seedu/address/model/recipe/CompletionTime.java

```diff
@@ -28,7 +30,7 @@ public CompletionTime(Integer completionTime) {
     }
 
     public static boolean isValidCompletionTime(Integer test) {
-        return (test > 0);
+        return (test > 0 && test < 525600);
```

Comment:
Should the upper bound be inclusive?

Reply:
I think its good to have an upper bound in case of testers putting in max_int size HAHAHHA @yusufaine

Reply:
1 year should be more than enough as an upper limit

Reply:
@blaketingyu oh i meant to ask if the condition should be `test <= 525600`, since people would test the boundary

Reply:
I think good to have a boundary uh if you ask me. It's like boundary value, equivalence partition stuff